website/pages/docs/agent/caching/index.mdx

```diff
@@ -160,11 +168,18 @@ $ curl \
 
 The top level `cache` block has the following configuration entries:
 
-- `use_auto_auth_token (bool: false)` - If set, the requests made to agent
+- `use_auto_auth_token (bool/string: false)` - If set, the requests made to agent
   without a Vault token will be forwarded to the Vault server with the
   auto-auth token attached. If the requests already bear a token, this
   configuration will be overridden and the token in the request will be used to
-  forward the request to the Vault server.
+  forward the request to the Vault server. If set to `"force"` Agent will use the
+  auto-auth token, overwriting the attached Vault token if set. 
+
+-> **Note:** Vault Agent does not require configuring auto-auth in order to use the
```

Comment:
Should this be removed, since `force` can be using w/o auto-auth?

Reply:
Yes! Removed in a65556b6e